res/openvpn-gui-res-no.rc

```diff
@@ -273,7 +273,7 @@ BEGIN
     IDS_NFO_RECONN_FAILED "Tilkobling til %s feilet."
     IDS_NFO_STATE_SUSPENDED "Status: Hvilemodus"
     IDS_ERR_READ_STDOUT_PIPE "Feil under lesning fra OpenVPN StdOut pipe."
-    IDS_ERR_CREATE_EDIT_LOGWINDOW "Opprettelse av RichEdit LogWindow feilet!!"
+    IDS_ERR_CREATE_EDIT_LOGWINDOW "Opprettelse av RichEdit LogWindow feilet!"
```

Comment:
Let us change that to Status Log Window. The RichEdit part is unnecessarily technical and probably from days when richedit20.dll was not supported (Windows 95/98 ?). Just guessing.